src/modules/TodoTask.test.js

```diff
@@ -0,0 +1,19 @@
+import TodoList from './TodoTask.js';
+
+describe('my TodoList class ', () => {
+  let kk;
+  beforeEach(() => {
+    kk = new TodoList();
+  });
+  test('add a new task', () => {
+    const lengthArry = kk.tasks.length;
+    kk.addTask('add the task');
+    expect(kk.tasks).toHaveLength(lengthArry + 1);
+  });
+  test('remove a task', () => {
+    kk.addTask('add the task');
+    const lengthArry = kk.tasks.length;
+    kk.removeList(1);
+    expect(kk.tasks).toHaveLength(lengthArry - 1);
+  });
+});
```

Comment:
- [ ] Kindly you have to add tests for the HTML and for the local storage as the requirements asked.
You will have to use mocks to test if add/delete functions add or remove exactly one `<li>` element to/from the list in the DOM and mock to test local storage.

Reply:
## Hi @keroloussamy 
Yeah, thanks for the review. I wanted to point out something, Our addTask and removeList functions are "Pure functions". Which means there are no side effects. Which means the functions don't manipulate the DOM or don't update the localStorage. I think

Reply:
Hello, @wuletawwonte it seems it's a mistake from me, I noticed that your functions are pure, but I wanted to let you try Mocks and test your Html and local storage, but your project is approved already so you are right. 
And sorry again 😄